UDx-container/vsdk-vertica

```diff
@@ -0,0 +1,165 @@
+
+# The user can override the OS selection by defining OSTAG in the
+# environment (mostly for testing)
+. /etc/os-release
+
+case $ID in
+    rhel) OSTAG=${OSTAG:=centos} ;;
+    ubuntu) OSTAG=${OSTAG:=ubuntu} ;;
+    *) OSTAG=${OSTAG:=centos} ;;
+esac
+case $OSTAG in
+    centos)
+        BINDIR=/usr/bin 
+        ALSO_IN_PATH=/bin
+        ;;
+    ubuntu)
+        BINDIR=/bin
+        ALSO_IN_PATH=/usr/bin
+        ;;
+esac
+
+V_VERSION=${VERTICA_VERSION:=11.0.0-0}
+IMAGE_NAME=verticasdk
+TAG=$OSTAG-$V_VERSION
+
+CONTAINER_NAME=verticasdk
+DVOL=verticasdk-data 
+
+VSDK_VOLUMES=""
+if [ "$VSDK_MOUNT"x != x ]; then
+    for dir in $VSDK_MOUNT; do
+        VSDK_VOLUMES="$VSDK_VOLUMES -v $dir:$dir"
+    done
+fi
+
+VSDK_ENV_FILE_OPTION=""
+if [ "$VSDK_ENV"x != x ]; then
+    VSDK_ENV_FILE_OPTION="--env-file $VSDK_ENV"
+fi
+
+CONTAINER_PATH=/opt/rh/rh-nodejs8/root/usr/bin
+CONTAINER_PATH=$CONTAINER_PATH:/opt/rh/llvm-toolset-7.0/root/usr/bin
+CONTAINER_PATH=$CONTAINER_PATH:/opt/rh/llvm-toolset-7.0/root/usr/sbin
+CONTAINER_PATH=$CONTAINER_PATH:/opt/rh/rh-git218/root/usr/bin
+CONTAINER_PATH=$CONTAINER_PATH:/opt/rh/devtoolset-7/root/usr/bin
+CONTAINER_PATH=$CONTAINER_PATH:/usr/lib64/ccache
+CONTAINER_PATH=$CONTAINER_PATH:${BINDIR}
+CONTAINER_PATH=$CONTAINER_PATH:${ALSO_IN_PATH}
+CONTAINER_PATH=$CONTAINER_PATH:/sbin
+CONTAINER_PATH=$CONTAINER_PATH:/usr/sbin
+CONTAINER_PATH=$CONTAINER_PATH:/usr/local/bin
+CONTAINER_PATH=$CONTAINER_PATH:/usr/local/sbin
+
+user_id=`id -u`
+HOSTNAME=vsdk-$USER
+
+function usage_exit() {
+    cat <<EOF
+Usage: $0 [-c cname] [-h] [-i img_name] [-t tag] [-v hostpath:containerdir] [-V docker-volume]
+Options are:
+ -c - specify container name (default is $CONTAINER_NAME)
+ -h - show help
+ -i image - specify image name (default is $IMAGE_NAME)
+ -t tag - specify the image tag (default is $TAG)
+ -v hostpath:containerdir - mount hostpath as containerdir in the 
+        container (in addition to the data docker volume)
+ -V volume - docker volume to use for the Vertica database (default is $DVOL)
+EOF
+    exit $1
+}
+
+while getopts "c:d:hi:r:t:v:V:" opt; do
+    case "$opt" in
+        c) CONTAINER_NAME="${OPTARG}"
+           ;;
+        h) usage_exit 0
+           ;;
+        i) IMAGE_NAME=${OPTARG}
+           ;;
+        t) TAG="${OPTARG}"
+           ;;
+        v) VFLAG="-v ${OPTARG}"
+           ;;
+        V) DVOL="${OPTARG}"
+           ;;
+        \?) echo "Invalid option: -$OPTARG" >&2
+            echo
+            usage_exit 1
+            ;;
+    esac
+done
+CURDIR=`/bin/pwd`
+
+echo "Starting container..."
+docker run \
+       --security-opt=seccomp:unconfined \
+       --privileged \
+       -e PATH=$CONTAINER_PATH \
+       -e vUID=$user_id \
+       -e vUSER=$USER \
+       -v "$HOME:$HOME:rw" \
+       -v ${CURDIR}:${CURDIR}:rw \
+       $VSDK_VOLUMES \
+       $VSDK_ENV_FILE_OPTION \
+       --mount type=volume,source=$DVOL,target=/data \
+       -p 5433:5433 \
+       -p 5444:5444 \
+       -d $VFLAG \
```

Comment:
Suggest we split this line like this:
```
-d \
$VFLAG \
```
At first glance it looks like you are providing $VFLAG as an value to the `-d` option.  But we are specifying two separate options here.